Pomile/Fast-Food-Fast
# #160918493: The admin should be able to add a food

#### What does this PR do?
This PR seeks to discuss changes around add a food  feature

#### Description of Task to be completed?
-Modify models
-Modify add a food item feature with persistent data
-Modify user feature

#### How should this be manually tested?
- Use postman to test api 
-on postman change the request type to post 
- Enter api url - http://localhost:8000/api/v1/fastFoods
-in the header section of api request call ensure that the following property is set
    content-type: application/json
    authorization: token
    isauth: true
- in the body section of the api request call 
    1. select raw and JSON(application/json)
    2.  enter the following in the body
         {
            "foodCategoryName": "Fries",
            "name": "Chicken and chips",
            "price": 2500,
            "description": "2 Chickens and a pack of chips",
            "quantity": 25,
            "expectedDeliveryTime": "45 min"
        }

- save
- ensure the server is running on your development machine
-send api request call

#### What are the relevant pivotal tracker stories?

#### Any background context you want to add?

#### Screenshots

![startdev](https://user-images.githubusercontent.com/28379439/47205511-08190e80-d37e-11e8-9a1f-9edc8d74740a.png)
![postman](https://user-images.githubusercontent.com/28379439/47205540-1f57fc00-d37e-11e8-96e7-bfaec9c90c5d.png)
